docs/understanding-docker.md

```diff
@@ -17,273 +17,282 @@ weight = -90
 
-This provides a layer of isolation: each aspect of a container runs in its own
-namespace and does not have access outside of it.
+These namespaces rovide a layer of isolation. Each aspect of a container runs
+in a separate namespace and its access is limited to that namespace.
 
-Some of the namespaces that Docker Engine uses on Linux are:
+Docker Engine uses namespaces such as the following on Linux:
 
  - **The `pid` namespace:** Process isolation (PID: Process ID).
  - **The `net` namespace:** Managing network interfaces (NET:
  Networking).
  - **The `ipc` namespace:** Managing access to IPC
  resources (IPC: InterProcess Communication).
- - **The `mnt` namespace:** Managing mount-points (MNT: Mount).
+ - **The `mnt` namespace:** Managing filesystem mount points (MNT: Mount).
  - **The `uts` namespace:** Isolating kernel and version identifiers. (UTS: Unix
 Timesharing System).
 
 ### Control groups
-Docker Engine on Linux also makes use of another technology called `cgroups` or control groups.
-A key to running applications in isolation is to have them only use the
-resources you want. This ensures containers are good multi-tenant citizens on a
-host. Control groups allow Docker Engine to share available hardware resources to
-containers and, if required, set up limits and constraints. For example,
-limiting the memory available to a specific container.
+Docker Engine on Linux also relies on another technology called _control groups_
+(`cgroups`). A cgroup limits an application to a specific set of resources.
+Control groups allow Docker Engine to share available hardware resources to
+containers and optionally enforce limits and constraints. For example,
+you can limit the memory available to a specific container.
 
 ### Union file systems
 Union file systems, or UnionFS, are file systems that operate by creating layers,
-making them very lightweight and fast. Docker Engine uses union file systems to provide
-the building blocks for containers. Docker Engine can make use of several union file system variants
-including: AUFS, btrfs, vfs, and DeviceMapper.
+making them very lightweight and fast. Docker Engine uses UnionFS to provide
+the building blocks for containers. Docker Engine can use multiple UnionFS variants,
+including AUFS, btrfs, vfs, and DeviceMapper.
 
 ### Container format
-Docker Engine combines these components into a wrapper we call a container format. The
-default container format is called `libcontainer`. In the future, Docker may
-support other container formats, for example, by integrating with BSD Jails
-or Solaris Zones.
+Docker Engine combines the namespaces, control groups, and UnionFS into a wrapper
+called a container format. The default container format is `libcontainer`. In
+the future, Docker may support other container formatsby integrating with
+technologies such as BSD Jails or Solaris Zones.
 
 ## Next steps
 Read about [Installing Docker Engine](installation/index.md#installation).
```

Comment:
Update "Next Steps" by making them bullet items, and add a link to the Getting Started tutorial:

* Read about [Installing Docker Engine](installation/index.md#installation).
* Get your feet wet with a beginner end-to-end tutorial [Get Started with Docker](/getstarted/index.md).
* Check out examples and deep dive topics in the [Docker Engine User Guide](/userguide/index.md).